benchmarking/benchmark_definitions/mlp_on_fashionmnist.py

```diff
@@ -22,33 +21,22 @@
 from syne_tune.remote.estimators import DEFAULT_CPU_INSTANCE
 
 
-def mlp_fashionmnist_default_params() -> Dict[str, Any]:
-    return {
-        "max_resource_level": 81,
-        "instance_type": DEFAULT_CPU_INSTANCE,
-        "num_workers": 4,
-        "dataset_path": "./",
-        "report_current_best": "False",
-    }
-
-
 def mlp_fashionmnist_benchmark(sagemaker_backend: bool = False, **kwargs):
-    params = mlp_fashionmnist_default_params()
     config_space = dict(
         _config_space,
-        dataset_path=params["dataset_path"],
-        epochs=params["max_resource_level"],
-        report_current_best=params["report_current_best"],
+        dataset_path="./",
+        epochs=81,
+        report_current_best="False",
     )
     _kwargs = dict(
-        script=Path(__file__).parent.parent.parent
+        script=Path(__file__).parent.parent
         / "training_scripts"
         / "mlp_on_fashion_mnist"
         / "mlp_on_fashion_mnist.py",
         config_space=config_space,
         max_wallclock_time=3 * 3600,  # TODO
-        n_workers=params["num_workers"],
-        instance_type=params["instance_type"],
+        n_workers=4,
+        instance_type=DEFAULT_CPU_INSTANCE,
```

Comment:
Likewise here, should these still be set through `params`?

Reply:
A few other instances of similar changes in `benchmarking/benchmark_definitions/lstm_wikitext2.py` and `benchmarking/benchmark_definitions/distilbert_on_imdb.py`

Reply:
See above.